python/paddle/fluid/tests/unittests/dygraph_to_static/ifelse_simple_func.py

```diff
@@ -0,0 +1,172 @@
+#   Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import paddle.fluid as fluid
+from paddle.fluid.dygraph.jit import dygraph_to_static_graph
+
+
+def dyfunc_with_if_else(x_v, label=None):
+    if fluid.layers.mean(x_v).numpy()[0] > 5:
+        x_v = x_v - 1
+    else:
+        x_v = x_v + 1
+    # plain if in python
+    if label is not None:
+        loss = fluid.layers.cross_entropy(x_v, label)
+        return loss
+    return x_v
+
+
+def dyfunc_with_if_else2(x, col=100):
+    row = 0
+    # plain if in python
+    if abs(col) > x.shape[-1]:
```

Comment:
After #PR22866 merged, `if abs(col) > x.shape[-1]:`will be transformde into `if abs(col) > fluid.layers.shape(x)[-1]:` which will not be a "plain if in python " and will be transformed into `fluid.layers.cond()`.

Reply:
Well, if `x.shape[-1]` was transformed into `fluid.layers.shape(x)[-1]`, that's ok for the case. It will be converted into `layers.cond`. I will remove the comment.